dm/dm/master/openapi.go

```diff
@@ -822,6 +823,123 @@ func (s *Server) DMAPIOperateTableStructure(c *gin.Context, taskName string, sou
 	}
 }
 
+// DMAPIBatchImportTaskConfig create task_config_template url is: (POST /api/v1/task_configs/batch_import).
+func (s *Server) DMAPIBatchImportTaskConfig(c *gin.Context) {
```

Comment:
this api is for this [[feature ](](https://docs.google.com/document/d/1OT70m8rVdMILETZLz0hdpIftP2Mo_jvH66mY9VAtj-E/edit?pli=1&disco=AAAAQ_UCKv4)url)